### DISCUSSION TITLE
ini/configuration guide missing a lot of options

### DISCUSSION THREAD

**Discussion by john-parton** [diasurgical/DevilutionX#4208] [General]
The configuration guide here: https://github.com/diasurgical/devilutionX/wiki/DevilutionX-diablo.ini-configuration-guide

Is missing a ton of options present in https://github.com/diasurgical/devilutionX/blob/1409e604f56698f7ea0c21b3820d20d4c6b1dba7/Source/options.cpp

I see "Show health values" and "Show mana values" here: https://github.com/diasurgical/devilutionX/blob/1409e604f56698f7ea0c21b3820d20d4c6b1dba7/Source/options.cpp#L760-L761

As well as "Quick cast" and a bunch of options for automatically picking up potions: https://github.com/diasurgical/devilutionX/blob/1409e604f56698f7ea0c21b3820d20d4c6b1dba7/Source/options.cpp#L833-L839

Should these all be added?

**Comment by qndel** [comment]
Feel free to update it.

**Comment by StephenCWills** [comment]
These options were all added between 1.3 and 1.4, were they not? It might be okay to update now since we're coming up on the release of 1.4, but I personally think this wiki page should be kept accurate with the most recent release version.

**Reply by john-parton** [reply]
Fair. I personally can't time my edits.

Would it be okay to add "New in version 1.X" next to them or something? I feel like that would be helpful all around.

Django does something like that in their docs: https://docs.djangoproject.com/en/4.0/ref/models/querysets/#default

**Reply by StephenCWills** [reply]
Hmm... For 1.4 we are introducing a Settings menu complete with brief descriptions for individual settings. I suspect there will be little, if any, reason to maintain this wiki page going forward. Since I'm not sure about the purpose or target audience at that point, I'm also not sure I can provide any guidance on what would be okay. Personally, I'd just leave the page alone for now and try to figure it out after the release.

**Reply by john-parton** [reply]
OK, cool. I didn't know about the settings menu. I'll see if I can build master and poke around.

**Comment by AJenbo** [comment]
It's a wiki so mostly community maintained, so no reason to have an issue for this. I'll convert it to a discussion though as that fair.

**Reply by john-parton** [reply]
Thanks. Some projects consider documentation issues like this as bugs. I'll not open issues like this in the future.

**Reply by AJenbo** [reply]
We would for any of the docs house in the files, like the various readmes and https://github.com/diasurgical/devilutionX/tree/master/docs
